java/client/src/main/java/glide/api/models/configuration/BackoffStrategy.java

```diff
@@ -12,6 +12,15 @@
  *
  * <p>Once the maximum value is reached, that will remain the time between retry attempts until a
  * reconnect attempt is successful. The client will attempt to reconnect indefinitely.
+ *
```

Comment:
how this is related to the pubsub

Reply:
I added example for new/changed confirations and then added to all rest for consistency.